wolfcrypt/src/asn.c

```diff
@@ -12388,7 +12388,15 @@ int GetFormattedTime(void* currTime, byte* buf, word32 len)
         hour = ts->tm_hour;
         mini = ts->tm_min;
         sec  = ts->tm_sec;
-        ret = XSNPRINTF((char*)buf, len,
+        #if defined(WOLF_C89)
+            if (len < 14) {
```

Comment:
Any way to avoid hard coded 14?

Reply:
none occured to me.  the ad hoc literal `14` is a function of the ad hoc format string, just adding up the literal `2`s in it, plus the "Z" and null.  you can't take `sizeof()` the format string obviously -- gives the wrong answer.

got any ideas?

(there's a reason `snprintf()` was added to the C standard...)